_protected/app/system/modules/payment/views/base/tpl/main/pay.tpl

```diff
@@ -16,7 +16,7 @@
     {if !$is_paypal AND !$is_stripe AND !$is_braintree AND !$is_2co AND !$is_skeerel AND !$is_ccbill}
         <p class="err_msg">{lang 'No Payment System Enabled!'}</p>
     {else}
-        {if $membership->enable == 1 AND $membership->price != 0}
```

Comment:
`$membership->price != 0` is to skip free memberships (since there are free, they shouldn't be shown up on this page).
https://github.com/Undefined-Variables/pH7-Social-Dating-CMS/blob/b67dcfb6b4d110e4066b96936feb050d71abce23/_protected/app/system/modules/payment/forms/MembershipForm.php#L86